src/test/java/edu/harvard/iq/dataverse/api/DataversesIT.java

```diff
@@ -1944,4 +1944,37 @@ public void testDeleteFeaturedItems() {
                 .body("message", equalTo("Can't find dataverse with identifier='thisDataverseDoesNotExist'"))
                 .statusCode(NOT_FOUND.getStatusCode());
     }
+
+    @Test
+    public void testUpdateInputLevelDisplayOnCreate() {
+        Response createUserResponse = UtilIT.createRandomUser();
+        String apiToken = UtilIT.getApiTokenFromResponse(createUserResponse);
+
+        Response createDataverseResponse = UtilIT.createRandomDataverse(apiToken);
+        createDataverseResponse.then().assertThat().statusCode(CREATED.getStatusCode());
+        String dataverseAlias = UtilIT.getAliasFromResponse(createDataverseResponse);
+
+        // Configure metadata blocks - disable inherit from root and set specific blocks
+        Response setMetadataBlocksResponse = UtilIT.setMetadataBlocks(
+                dataverseAlias, 
+                Json.createArrayBuilder().add("socialscience"), 
+                apiToken);
+        setMetadataBlocksResponse.then().assertThat()
+                .statusCode(OK.getStatusCode());
+
+        // Verify initial state
+        Response listMetadataBlocks = UtilIT.listMetadataBlocks(dataverseAlias, true, true, apiToken);
+        listMetadataBlocks.then().assertThat()
+                .statusCode(OK.getStatusCode())
+                .body("data.size()", equalTo(1))
```

Comment:
I'm on 97b454da06 and I'm getting this failure...

```
[ERROR] Failures: 
[ERROR]   DataversesIT.testUpdateInputLevelDisplayOnCreate:1969 1 expectation failed.
JSON path data.size() doesn't match.
Expected: <1>
  Actual: <0>
```

... when I execute just DataversesIT (`mvn test -Dtest=DataversesIT`) on a freshly spun up Docker instance.

Reply:
This is what the output looks like:

```
{
    "status": "OK",
    "data": [
        
    ]
}
```

(Locally, I added `listMetadataBlocks.prettyPrint();`.)